crates/egui/src/viewport.rs

```diff
@@ -799,6 +817,48 @@ pub enum WindowLevel {
     AlwaysOnTop,
 }
 
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum WindowType {
```

Comment:
Better to re-export `winit::platform::x11::XWindowType` instead, no?

Reply:
Yeah I can do that. Do you think I should change it to take multiple
XWindowType's too? I'm not sure I understand the point of that, but it is supported for some reason.

Reply:
done! It does unfortunately make egui depend on winit directly, thoughts? I guess the alternative is an extension trait in egui_winit?

Reply:
Oh no, my bad 🤦 
I thought this code was in `egui-winit`
Yeah, we can't have `egui` depend on `winit`